contrib/rust-analyzer/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "rust-analyzer"
+_pver = "2023-08-14"
+pkgver = _pver.replace("-", ".")
```

Comment:
use the version string for pkgver (i.e. `pkgver = "2023.08.14"`) and then use `{pkgver.replace('.', '-')}` in `source`

Reply:
done